lib/commons/dom/create-grid.js

```diff
@@ -267,11 +267,11 @@ function createStackingOrder(vNode, parentVNode, nodeIndex) {
   if (!isStackingContext(vNode, parentVNode)) {
     if (vNode.getComputedStylePropertyValue('position') !== 'static') {
       // Put positioned elements above floated elements
-      stackingOrder.push(POSITION_STATIC_ORDER);
+      stackingOrder.push(createContext(POSITION_STATIC_ORDER, vNode));
```

Comment:
Adding virtual node info to the stacking order allows me to use that virtual node to create the stacks for the node.